debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -718,14 +721,25 @@ public void startKeepAlive(ExecutorService service) {
                             try {
                                 LOGGER.trace("Forcing status update with replication stream");
                                 stream.forceUpdateStatus();
-                                metronome.pause();
                             }
                             catch (Exception exp) {
-                                throw new RuntimeException("received unexpected exception will perform keep alive", exp);
+                                LOGGER.error(
+                                        "received unexpected exception will perform keep alive",
+                                        exp);
+                                keepAliveException = exp;
+                            }
+
+                            try {
+                                metronome.pause();
+                            }
+                            catch (InterruptedException e) {
+                                LOGGER.warn(
+                                        "Keep alive thread has already be shutdown.");
```

Comment:
This formatting for `LOGGER` does not really conform to the remainder of the code base. 
Would you please align those to be on a single line each, thanks!.